docs/releases/2.0.txt

```diff
@@ -143,6 +143,10 @@ Database backends
 * Oracle database introspection now works with auto fields, but only for newly
   created.
 
+* Added support for the :class:`~django.db.models.AutoField`,
+  :class:`~django.db.models.BigAutoField` fields as Oracle Identity Columns
```

Comment:
- are there any concerns about having a mix of identity and non-identity columns in a project?
> I don't think so.
-  should we provide a way to update existing columns to identity columns?
> There is no way to update existing columns to identity columns. Identity cannot be "attached" to existing columns. Theoretical scenario: drop PK constraint, add new identity column, copy data from old PK, adjust sequence value, drop old PK. But this scenario won't work for columns referenced by FKs (*ORA-02273: this unique/primary key is referenced by some foreign keys*), hence it's almost always useless. 
- explain the benefits of this or link "identity columns" to the Oracle docs?
> The biggest benefit is performance increase (see [article](https://oracle-base.com/articles/12c/identity-columns-in-oracle-12cr1)). The Oracle docs on this subject is laconic, but we can add a link to the official docs http://docs.oracle.com/database/121/DRDAA/migr_tools_feat.htm#DRDAA109.